lizmap/modules/presentation/urls.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<suburls xmlns="http://jelix.org/ns/suburls/1.0">
+   
```

Comment:
you should declare url of each action here, and create a `install/configurator.php` to declare the url (see example into some external modules, like wps)